README.md

```diff
@@ -13,30 +13,23 @@
 - [📋 Requirements](#-requirements)
 - [✅ Getting started](#-getting-started)
   - [🧰 Installing `ilab`](#-installing-ilab)
-    - [To install with no GPU acceleration and PyTorch without CUDA bindings](#to-install-with-no-gpu-acceleration-and-pytorch-without-cuda-bindings)
-    - [To install with AMD ROCm](#to-install-with-amd-rocm)
-    - [To install with Apple Metal on M1/M2/M3 Mac](#to-install-with-apple-metal-on-m1m2m3-mac)
-    - [To install with Nvidia CUDA](#to-install-with-nvidia-cuda)
-    - [Example output](#example-output)
-    - [Bash (version 4.4 or newer)](#bash-version-44-or-newer)
-    - [Zsh](#zsh)
-    - [Fish](#fish)
+    - [Install with no GPU acceleration and PyTorch without CUDA bindings](#install-using-pytorch-without-cuda-bindings-and-no-gpu-acceleration)
+    - [Install with AMD ROCm](#install-with-amd-rocm)
+    - [Install with Apple Metal on M1/M2/M3 Macs](#install-with-apple-metal-on-m1m2m3-macs)
+    - [Install with Nvidia CUDA](#install-with-nvidia-cuda)
   - [🏗️ Initialize `ilab`](#️-initialize-ilab)
-    - [Example output](#example-output-1)
-    - [Example output](#example-output-2)
   - [📥 Download the model](#-download-the-model)
   - [🍴 Serving the model](#-serving-the-model)
   - [📣 Chat with the model (Optional)](#-chat-with-the-model-optional)
 - [💻 Creating new knowledge or skills and training the model](#-creating-new-knowledge-or-skills-and-training-the-model)
   - [🎁 Contribute knowledge or compositional skills](#-contribute-knowledge-or-compositional-skills)
   - [📜 List and validate your new data](#-list-and-validate-your-new-data)
   - [🚀 Generate a synthetic dataset](#-generate-a-synthetic-dataset)
-    - [Example output](#example-output-3)
-  - [👩‍🏫 Train the model](#-train-the-model)
+  - [👩‍🏫 Training the model](#-training-the-model)
     - [Train the model locally on Linux](#train-the-model-locally-on-linux)
-    - [Train the model locally on an M-series Mac](#train-the-model-locally-on-an-m-series-mac)
-    - [Training the model locally with GPU acceleration](#training-the-model-locally-with-gpu-acceleration)
-    - [Training the model in the cloud](#training-the-model-in-the-cloud)
+    - [Train the model locally on M-series Macs](#train-the-model-locally-on-an-m-series-mac)
+    - [Train the model locally with GPU acceleration](#training-the-model-locally-with-gpu-acceleration)
+    - [Train the model in the cloud](#training-the-model-in-the-cloud)
```

Comment:
This table-of-contents is automatically generated and updated by a vscode plugin. (Or it is for me!)

This last line at least appears to change the ToC but not the actual heading, so it's going to get changed back when someone else edits the file.

Reply:
Hmm the toc does not automatically update for me like that.
I do see theres a inconstancy in that though so I can fix that!

Reply:
@russellb just updated!

Reply:
@kelbrown20 if you use vscode, try this: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one